int32 id
string name

# 1: vehicle
# 2: pedestrian
# 3: misc
int8 type

geometry_msgs/Pose pose

float32 speed

float32 length
float32 width
float32 height

float32 wheel_angle